### ISSUE TITLE
Model Tuning

### ISSUE THREAD

**Issue by collinarnett** [collinarnett/protein_gan#7]
Currently there are a few hyper parameters that aren't mentioned in the paper that need to be optimized:
- batch_size
- num_epochs
- random_seed
Batch size heavily affects training and num_epochs can be tuned after focusing on batch size as a hyperparameter. Random Seed randomly affects training so it will remain static at 666 until the very end.

**Comment by collinarnett** [comment]
75c402dde337061f44189a1b32328763dcfafd59 
added some new changes to better reflect paper for better training stability.

**Comment by collinarnett** [comment]
I've tried to implement changes to decrease loss of the generator but I can't seem to hack it without changing the entire architecture. One sided label smoothing definitely helps but is not perfect and introducing noice every 20 steps does not improve model learning. 
![image](https://user-images.githubusercontent.com/38230482/82866950-e687ed80-9ef7-11ea-9a6d-b4374c6f1a85.png)

**Comment by collinarnett** [comment]
Going to leave this now since the paper is now fully replicated.
